pkg/kubelet/dockershim/network/hostport/hostport_manager.go

```diff
@@ -394,3 +394,18 @@ func filterChains(chains map[utiliptables.Chain]string, filterChains []utiliptab
 		delete(chains, chain)
 	}
 }
+
+func getPodFullName(pod *PodPortMapping) string {
```

Comment:
simplifying code review: these helper methods moved from the removed hostport files (see below). These were shared